pyramid_zipkin/request_helper.py

```diff
@@ -28,23 +28,7 @@ class ZipkinAttrs(namedtuple(
 
 def generate_random_64bit_string():
     """Returns a 64 bit ascii encoded string"""
```

Comment:
Nit, but technically it's returning a UTF-8 encoded string. You can change the docstring, or you could use the ASCII codec to decode.